bld/CMakeLists.txt

```diff
@@ -66,11 +66,13 @@ endif()
 
 if (CMAKE_VERSION VERSION_LESS "3.1")
```

Comment:
It would be better to change this to a macro, to avoid repeating the stuff across all CMakeList.txt files.  It used to be that this was a macro, named compileAsC99, that might be good to go back to.  I am surprised though that this does not happen on Ubuntu (with gcc 4.8, 6.2.0.*).  What was the compiler you used on Debian?  Another option, which I may prefer is to unset the c standard before including libwebsockets and re-setting it after (like done with warnings).  Might not work though, cmake is finicky with these variables...